### PR TITLE
[SSL] Apply tabs partially + break out encryption options into separate pages

### PR DESCRIPTION
**Pull request by kodster28**

### CODE REVIEW

File: content/ssl/origin-configuration/ssl-modes/_index.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+---
+pcx_content_type: reference
+title: Encryption modes
+weight: 2
+---
+
+# Encryption modes
```
Comment (kodster28) [inline content/ssl/origin-configuration/ssl-modes/_index.md:7]: This is content from the previous SSL modes page.

File: content/ssl/origin-configuration/ssl-modes/flexible.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+---
```
Comment (kodster28) [inline content/ssl/origin-configuration/ssl-modes/flexible.md:1]: This is just breaking out content from https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/ onto a separate page

File: content/ssl/origin-configuration/ssl-modes/full-strict.md [hunk 3]
```diff
@@ -0,0 +1,31 @@
+---
+pcx_content_type: reference
+title: Full (strict)
+weight: 4
+meta:
+    title: Full (strict) - SSL/TLS encryption modes
+---
+
```
Comment (kodster28) [inline content/ssl/origin-configuration/ssl-modes/full-strict.md:8]: This is just breaking out content from https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/ onto a separate page

File: content/ssl/origin-configuration/ssl-modes/full.md [hunk 4]
```diff
@@ -0,0 +1,29 @@
+---
+pcx_content_type: reference
+title: Full
+weight: 3
+meta:
+    title: Full - SSL/TLS encryption modes
+---
+
```
Comment (kodster28) [inline content/ssl/origin-configuration/ssl-modes/full.md:8]: This is just breaking out content from https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/ onto a separate page

File: content/ssl/origin-configuration/ssl-modes/off.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
+---
+pcx_content_type: reference
+title: Off (no encryption)
+weight: 1
+meta:
+    title: Off - SSL/TLS encryption modes
+---
+
```
Comment (kodster28) [inline content/ssl/origin-configuration/ssl-modes/off.md:8]: This is just breaking out content from https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/ onto a separate page

File: content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
+---
+pcx_content_type: reference
+title: Strict (SSL-Only Origin Pull)
+weight: 5
+meta:
+    title: Strict (SSL-Only Origin Pull) - SSL/TLS encryption modes
+---
+
```
Comment (kodster28) [inline content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md:8]: This is just breaking out content from https://developers.cloudflare.com/ssl/origin-configuration/ssl-modes/ onto a separate page

File: content/ssl/saas.md [hunk 7]
```diff
@@ -1,13 +1,9 @@
...
-For example, a customer may want to use their vanity domain `app.customer.com` to point to an application hosted on your Cloudflare zone `service.saas.com`.
-
-SSL for SaaS is now part of the broader feature, [Cloudflare for SaaS](/cloudflare-for-saas/). As of July 11, 2022, the documentation has been relocated to [Cloudflare for SaaS - SSL](/cloudflare-for-saas/security/certificate-management/).
\ No newline at end of file
+external_link: /cloudflare-for-saas/
+weight: 8
+_build:
+  publishResources: false
+  render: never
+---
```
Comment (kodster28) [inline content/ssl/saas.md:9]: This is changing a navigation page into a direct external link w/in the side navigation.

File: content/ssl/edge-certificates/additional-options/automatic-https-rewrites.md [hunk 8]
```diff
@@ -22,16 +22,27 @@ For security reasons, this feature will run on URLs pointing to `localhost` if t
...
 
-If a third-party domain supports HTTPS and is not rewritten automatically, you can manually change those links to relative links or HTTPS links. Alternatively, you can ask the third-party domain owner to submit their site for inclusion in the HTTPS Everywhere rulesets, which [accept pull requests on GitHub](https://github.com/EFForg/https-everywhere/). For more information on how to fix mixed content errors, see [Troubleshooting mixed content errors](https://support.cloudflare.com/hc/articles/200170476).
+If a third-party domain supports HTTPS and is not rewritten automatically, you can manually change those links to relative links or HTTPS links. Alternatively, you can ask the third-party domain owner to submit their site for inclusion in the HTTPS Everywhere rulesets, which [accept pull requests on GitHub](https://github.com/EFForg/https-everywhere/). For more information on how to fix mixed content errors, see [Troubleshooting mixed content errors](https://support.cloudflare.com/hc/articles/200170476).
```
Comment (ranbel) [inline content/ssl/edge-certificates/additional-options/automatic-https-rewrites.md]:
```suggestion
If a third-party domain supports HTTPS and is not rewritten automatically, you can manually change those links to relative links or HTTPS links. Alternatively, you can ask the third-party domain owner to submit their site for inclusion in the HTTPS Everywhere rulesets, which [accept pull requests on GitHub](https://github.com/EFForg/https-everywhere/). For more information on how to fix mixed content errors, refer to [Troubleshooting mixed content errors](https://support.cloudflare.com/hc/articles/200170476).
```

File: content/ssl/edge-certificates/additional-options/opportunistic-encryption.md [hunk 9]
```diff
@@ -11,11 +11,22 @@ Use HTTPS when both strong encryption and authentication are required. HTTP Oppo
...
+To enable Opportunistic Encryption in the dashboard:
+ 
 1.  Log in to your [Cloudflare account](https://dash.cloudflare.com) and go to a specific domain.
 2.  Navigate to **SSL/TLS** > **Edge Certificates**.
 3.  For **Opportunistic Encryption**, switch the toggle to **On**.
+ 
+{{</tab>}}
+{{<tab label="api" no-code="true">}}
+ 
+To adjust your **Opportunistic Encryption** settings with the API, send a [`PATCH`](https://api.cloudflare.com/#zone-settings-change-opportunistic-encryption-setting) request with the `value` parameter set to your desired setting (`"on"` or `"off"`).
```
Comment (ranbel) [inline content/ssl/edge-certificates/additional-options/opportunistic-encryption.md]:
```suggestion
To adjust your Opportunistic Encryption settings with the API, send a [`PATCH`](https://api.cloudflare.com/#zone-settings-change-opportunistic-encryption-setting) request with the `value` parameter set to your desired setting (`"on"` or `"off"`).
```

Comment (ranbel) [reply content/ssl/edge-certificates/additional-options/opportunistic-encryption.md]: (was unbolded for the dashboard instructions)

File: content/ssl/edge-certificates/additional-options/tls-13.md [hunk 10]
```diff
@@ -19,11 +19,25 @@ By turning on the TLS 1.3 feature, traffic to and from your website will be serv
...
+To enable TLS 1.3 in the dashboard:
 
 1.  Log in to your [Cloudflare account](https://dash.cloudflare.com) and go to a specific domain.
 2.  Navigate to **SSL/TLS** > **Edge Certificates**.
 3.  For **TLS 1.3**, switch the toggle to **On**.
+ 
+{{</tab>}}
+{{<tab label="api" no-code="true">}}
+ 
+To adjust your **TLS 1.3** settings with the API, send a [`PATCH`](https://api.cloudflare.com/#zone-settings-change-tls-1.3-setting) request with the `value` parameter set to your desired setting (`"on"` or `"off"`).
```
Comment (ranbel) [inline content/ssl/edge-certificates/additional-options/tls-13.md]:
```suggestion
To adjust your TLS 1.3 settings with the API, send a [`PATCH`](https://api.cloudflare.com/#zone-settings-change-tls-1.3-setting) request with the `value` parameter set to your desired setting (`"on"` or `"off"`).
```

File: content/ssl/edge-certificates/custom-certificates/caa-records.md [hunk 11]
```diff
@@ -57,7 +60,13 @@ To add a CAA record:
...
 8.  Click **Save**.
 9.  Repeat for each CA associated with your domain.
-
-Once you have finished creating all the records, you can review them in the list of records appearing under the DNS Records panel.
-
+ 
+{{</tab>}}
+{{<tab label="api" no-code="true">}}
+ 
 To create these records via the API, use this [POST endpoint](https://api.cloudflare.com/#dns-records-for-a-zone-create-dns-record).
```
Comment (ranbel) [inline content/ssl/edge-certificates/custom-certificates/caa-records.md]:
```suggestion
To create a CAA record via the API, use this [POST endpoint](https://api.cloudflare.com/#dns-records-for-a-zone-create-dns-record).
```

File: content/ssl/origin-configuration/ssl-modes/off.md [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+# Off - SSL/TLS encryption modes
+
+Setting your encryption mode to **Off (not recommended)** redirects any HTTPS request to plaintext HTTP.
+
+![With an encryption mode of Off, your application does not encrypt traffic between the visitor and Cloudflare or between Cloudflare and your server.](/ssl/static/ssl-encryption-mode-off.png)
+
+## Use when
+
+Cloudflare does **not recommend** setting your encryption mode to **Off**.
```
Comment (ranbel) [inline content/ssl/origin-configuration/ssl-modes/off.md]:
```suggestion
Cloudflare does not recommend setting your encryption mode to **Off**.
```

File: content/ssl/origin-configuration/ssl-modes/off.md [hunk 13]
```diff
@@ -0,0 +1,35 @@
...
+
+## Required setup
+
+There is no required set up for this option.
```
Comment (ranbel) [inline content/ssl/origin-configuration/ssl-modes/off.md]:
```suggestion
There is no required setup for this option.
```

File: content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md [hunk 14]
```diff
@@ -0,0 +1,29 @@
...
+# Strict (SSL-Only Origin Pull) - SSL/TLS encryption modes
+
+_This method is only available for Enterprise zones._
```
Comment (ranbel) [inline content/ssl/origin-configuration/ssl-modes/ssl-only-origin-pull.md]:
```suggestion
{{<Aside type="note">}}
This method is only available for Enterprise zones.
{{</Aside>}}
```

File: content/ssl/origin-configuration/ssl-tls-recommender.md [hunk 15]
```diff
@@ -18,11 +18,22 @@ For more background, refer to the [introductory blog post](https://blog.cloudfla
...
+To enable SSL/TLS recommendations in the dashboard:
 
 1.  Log in to the [Cloudflare dashboard](https://dash.cloudflare.com) and select your account and application.
 2.  Navigate to **SSL/TLS**.
 3.  For **SSL/TLS Recommender**, switch the toggle to **On**.
+ 
+{{</tab>}}
+{{<tab label="api" no-code="true">}}
+ 
+To adjust your **SSL/TLS Recommender** enrollment with the API, send a [`PATCH`](https://api.cloudflare.com/#zone-settings-change-ssl/tls-recommender-enrollment) request with the `enabled` parameter set to your desired setting (`true` or `false`).
```
Comment (ranbel) [inline content/ssl/origin-configuration/ssl-tls-recommender.md]:
```suggestion
To adjust your SSL/TLS Recommender enrollment with the API, send a [`PATCH`](https://api.cloudflare.com/#zone-settings-change-ssl/tls-recommender-enrollment) request with the `enabled` parameter set to your desired setting (`true` or `false`).
```

**Review by ranbel — APPROVED** [review]

**Pull request merged**